### DISCUSSION TITLE
Fancy Tab not answering to colors.tab_bar.background?

### DISCUSSION THREAD

**Discussion by mobius3** [wezterm/wezterm#1482] [Q&A] [answered]
Is there a configuration change on how to change tab background color? It seems to be picking up my system theme, which is nice, but changing `colors.tab_bar.background` does nothing. `colors.tab_bar.active_tab` configs do work though so I'm wondering if I should open an issue this or if it's just a config change that I missed somewhere.

Thanks!

**Comment by wez** [comment, marked as answer]
I've updated the docs at: https://wezfurlong.org/wezterm/config/appearance.html#native-fancy-tab-bar-appearance

**Reply by mobius3** [reply]
Thaaanks!
